src/content/ui-api/observability/prometheus/volume-size/index.md

```diff
@@ -0,0 +1,87 @@
+---
+linkTitle: Prometheus Volume Size
+title: Prometheus VolumeSize
+description: A guide explaining how to size the Prometheus persistent volume
+menu:
+  main:
+    identifier: uiapi-observability-prometheusvolumesize
+    parent: uiapi-observability-prometheus
+user_questions:
+  - How can I downsize the persistent volume?
+  - How do I expand the Prometheus volume?
+  - Why would I like to resize the Prometheus persistent volume ?
+aliases:
+  - /observability/prometheus/volume-size
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+---
+
+In this document you will learn how to size the Prometheus persistent volume.
+
+__Warning:__ 
+* The volume expansion on OpenStack clusters is currently not working properly
+* Downsizing the persistent volume will cause a data loss
+* This feature and the documentation is quite new, so do not hesitate to ask for support or help us improve this documentation.
+
+## Introduction to persistent volume
+Each Prometheus instance needs a persistent volume to store the recorded metrics.
```

Comment:
```suggestion
Prometheus uses a persistent volume to store its recorded metrics and for historical reasons, we decided to define the needed capacity to 100Gi per cluster (Workload and management clusters alike).
```

Reply:
I would Remove the introduction and write this before the warnings. The rest of the introduciotn seems redundant to me